backend/testfiles/execution/stdlib/json.dark

```diff
@@ -163,6 +236,19 @@ Json.parse<Float> "{ \"key\": 42 }" = PACKAGE.Darklang.Stdlib.Result.Result.Erro
   "Can't currently parse this type/value combination"
 
 Json.parse<Float> "\"42\n\"" = PACKAGE.Darklang.Stdlib.Result.Result.Error "not JSON"
+
+// are these exoected to be not JSON?
+Json.parse<Float> "000000.9" = PACKAGE.Darklang.Stdlib.Result.Result.Error "not JSON"
```

Comment:
Weird! I've never encountered this, but sites like https://jsonlint.com/ seem to also be reporting this as invalid JSON.